contracts/serverless-v2/Jobs.sol

```diff
@@ -322,11 +346,58 @@ contract Jobs is
             )
         );
         bytes32 digest = keccak256(abi.encodePacked("\x19\x01", DOMAIN_SEPARATOR, hashStruct));
-        signer = digest.recover(_signature);
+        address signer = digest.recover(_signature);
 
         executors.allowOnlyVerified(signer, _executor);
     }
 
+    function _transferRewardPayout(
+        uint256 _jobId,
+        uint256 _outputCount,
+        address _executor
+    ) internal {
+        uint256 totalTime = jobs[_jobId].firstOutputExecutionTime;
+        // for first output
+        if(_outputCount == 1) {
+            // transfer payout to executor
+            TOKEN.safeTransfer(_executor, (totalTime * EXECUTOR_FEE_PER_MS * 2) / 3);
+            // TODO: is payment pool the jobs contract itself?
```

Comment:
Actually, there will be two tokens now, slashing is in pond, and job payment in the USDC. So, account address can be same, but token would be different.

Reply:
Yes, that is it. But both tokens aren't stored in any of the contracts. Since escrow deposit and reward payout (USDC) happens in Relay and Jobs contracts, whereas the stake deposit and slashing happens in the Gateways and Executors contracts.

Reply:
If payment pool refers to the Jobs contract, then the commented line L366 isn't required - 
`// TOKEN.safeTransfer(address(this), STAKING_REWARD_PER_MS);`

Reply:
Okay, right now, assume USDC payment pool is Job contract address itself. We will add mechanism later, which will consume the payment pool (admin would be allowed to allocate it to stakers).